var/spack/repos/builtin/packages/elfutils/package.py

```diff
@@ -50,6 +53,14 @@ class Elfutils(AutotoolsPackage):
     # elfutils with gcc, and then link it to clang-built libraries.
     conflicts('%clang')
 
+    # Elfutils uses -Wall and we don't want to fail the build over a
+    # stray warning.
+    def patch(self):
+        files = glob.glob(os.path.join('*', 'Makefile.in'))
+        filter_file('-Werror', '', *files)
+
+    flag_handler = AutotoolsPackage.build_system_flags
```

Comment:
I realize I've been away from Spack for quite some time, but what does this line do?

Reply:
@adamjstewart this makes it so that compiler flags set in `compilers.yaml` or on the command line will be passed as arguments to the build system (to avoid cases where the build system chooses flag values that conflict with compilers.yaml).